docs/reference/ilm/apis/slm-api.asciidoc

```diff
@@ -110,28 +110,56 @@ To update an existing policy, simply use the put snapshot lifecycle policy API
 with the same policy id as an existing policy.
 
 [[slm-api-get]]
-=== Get Snapshot Lifecycle Policy API
+=== Get snapshot lifecycle policy API
+++++
+<titleabbrev>Get snapshot lifecycle policy</titleabbrev>
+++++
 
-Once a policy is in place, you can retrieve one or more of the policies using
-the get snapshot lifecycle policy API. This also includes information about the
-latest successful and failed invocation that the automatic snapshots have taken.
+Returns information
+about one or more snapshot lifecycle policies.
```

Comment:
What is up with the ~50 character line wrapping?

Reply:
I think it makes it much harder to review, but that may be just my personal preference.

Reply:
Thanks for the feedback!